setuptools/monkey.py

```diff
@@ -130,7 +130,7 @@ def patch_func(replacement, target_mod, func_name):
 
 
 def get_unpatched_function(candidate):
-    return getattr(candidate, 'unpatched')
+    return getattr(candidate, 'unpatched', None)
```

Comment:
I thought again about this one, and I think the error message would be cleaner if we just try to access the `unpatched` attribute directly...

If `None` gets retuned, it might be more difficult to trace the root cause of the problem when debugging.

I will submit a follow up commit.